Client/Frontend/Widgets/SiteTableViewController.swift

```diff
@@ -164,6 +161,17 @@ class SiteTableViewController: UIViewController, UITableViewDelegate, UITableVie
         }
     }
 
+    private func addTableView() {
```

Comment:
```suggestion
    private func setupView() {
```
As per my comment above. :)